modules/ROOT/pages/changes-deprecations-removals.adoc

```diff
@@ -0,0 +1,601 @@
+|Removed without replacement.
+
+|link:{neo4j-docs-base-uri}/operations-manual/5/configuration/configuration-settings#config_dbms.kubernetes.service_port_name[dbms.kubernetes.service_port_name] label:enterprise[Enterprise Edition]
+|Removed without replacement.
+
+|link:{neo4j-docs-base-uri}/operations-manual/5/configuration/configuration-settings#config_initial.dbms.database_allocator[initial.dbms.database_allocator] label:enterprise[Enterprise Edition]
+|Removed  without replacement.
+
+|link:{neo4j-docs-base-uri}/operations-manual/5/configuration/configuration-settings#config_server.cluster.catchup.connect_randomly_to_server_group[server.cluster.catchup.connect_randomly_to_server_group] label:enterprise[Enterprise Edition] label:dynamic[]
+|Removed without replacement. +
+Instead use xref:configuration/configuration-settings.adoc#config_server.cluster.catchup.connect_randomly_to_server_tags[`server.cluster.catchup.connect_randomly_to_server_tags`]
+
+|link:{neo4j-docs-base-uri}/operations-manual/5/configuration/configuration-settings#config_server.discovery.advertised_address[server.discovery.advertised_address] label:enterprise[Enterprise Edition]
+|Removed without replacement. +
+Instead use the value of xref:configuration/configuration-settings.adoc#config_server.cluster.advertised_address[`server.cluster.advertised_address`].
+
+|link:{neo4j-docs-base-uri}/operations-manual/5/configuration/configuration-settings#config_server.discovery.listen_address[server.discovery.listen_address] label:enterprise[Enterprise Edition]
+|Removed without replacement. +
+Instead use the value of xref:configuration/configuration-settings.adoc#config_server.cluster.listen_address[`server.cluster.listen_address`].
+
+|link:{neo4j-docs-base-uri}/operations-manual/5/configuration/configuration-settings#config_server.groups[server.groups] label:enterprise[Enterprise Edition]
+|Replaced by xref:configuration/configuration-settings.adoc#config_server.tags[`initial.server.tags`].
+
+|link:{neo4j-docs-base-uri}/operations-manual/5/configuration/configuration-settings#config_server.memory.off_heap.block_cache_size[server.memory.off_heap.block_cache_size]
+|Removed without replacement.
+
+|link:{neo4j-docs-base-uri}/operations-manual/5/configuration/configuration-settings#config_server.memory.off_heap.max_cacheable_block_size[server.memory.off_heap.max_cacheable_block_size]
+|Removed without replacement.
+
+|link:{neo4j-docs-base-uri}/operations-manual/5/configuration/configuration-settings#config_server.memory.off_heap.transaction_max_size[server.memory.off_heap.transaction_max_size]
+|Removed without replacement.
+
+|===
+====
+
+
+==== Changes to default values of configuration settings
+
+These changes affect only new installations or upgrades that replace existing configuration files.
```

Comment:
I understand it as the upgrades can replace config files. Maybe this is what you want to say?
```suggestion
These changes affect only new installations or upgrades that involve replacing existing configuration files.
```